packages/utils-compat-data/README.md

```diff
@@ -0,0 +1,172 @@
+# Utils compat data (`@hint/utils-compat-data`)
+
+This package allows you to easily query the data from [mdn-browser-compat-data][]
+to know about the support status of a feature.
+
+The type of information you can query is explained below:
+
+## API
+
+The supported datasets are CSS and HTML. Most of the exposed functionality
+uses a `FeatureQuery` to indicate the type of information you are querying.
+You can retrieve information for:
+
+* Attributes:
+
+```ts
+export type AttributeQuery = {
+    attribute: string;
+    element?: string;
+    value?: string;
+};
+```
+
+* Elements:
+
+```ts
+export type ElementQuery = {
+    element: string;
+};
+```
+
+* Declarations:
+
+```ts
+export type DeclarationQuery = {
+    property: string;
+    value?: string;
+};
+```
+
+* Rules:
+
+```ts
+export type RuleQuery = {
+    rule: string;
+};
+```
+
+* Selectors:
+
+```ts
+export type SelectorQuery = {
+    selector: string;
+};
+```
+
+Most of the functions accept a `FeatureQuery` and a list of browser
+ids to check. This package accepts two different formats of browsers
+ids: browserslist or MDN.
+
+The friendly names and ids for each one are as follows:
+
+| Browser name | [Browserslist][bl browsers] | [MDN][mdn browsers] |
+| ------------ | ------------ | --------- |
+| Chrome       | chrome       | chrome |
+| Chrome Android | and_chr    | chrome_android |
+| Edge         | edge         | edge |
+| Firefox      | ff           | firefox |
+| Firefox Android | and_ff    | firefox_android |
+| Internet Explorer | ie      | ie |
+| Opera        | opera        | opera |
+| Opera Android | op_mobile   | opera_android |
+| QQ Android   | and_qq       | qq_android |
+| Safari       | safari       | safari |
+| Safari iOS   | ios_saf      | safari_ios |
+| Samsung Internet | samsung  | samgsunginternet_android |
+| UC Android   | and_uc       | uc_android |
+| Webview Android | android   | webview_android |
+
+You can check the next sections for example on how to use `FeatureQuery`s
+and list of browsers.
+
+### `getFriendlyName`
+
+Get the friendly name of a browser from an id.
+
+```js
+import { getFriendlyName } from '@hint/utils-compat-data';
+
+console.log(getFriendlyName('and_ff')); // "Firefox Android"
+console.log(getFriendlyName('ie')); // "Internet Explorer"
+```
+
+### `getUnsupported`
```

Comment:
I think we're missing `getSupported` from this list (basically the opposite of `getUnsupported`).

Reply:
Added